src/components/Carousel/Carousel.stories.tsx

```diff
@@ -310,7 +311,9 @@ Scroller.args = {
             width: '100%',
           }}
         >
-          {item.name}
+          <Tooltip content={item.name} portal>
```

Comment:
Suggest copy/pasting the Tooltip with truncation here for a better E2E example of a typical Card text. Maybe via a quick concatenation of the item and index with some Lorem Ipsum - or the same text from the [Truncation Detection](https://github.com/EightfoldAI/octuple/blob/main/src/components/Tooltip/Tooltip.stories.tsx#L131) story.